controls/radtreelistview/radtreeliestview-getting-started.md

```diff
@@ -302,15 +377,30 @@ By default the __RadTreeListView__ will auto-generate the columns for your data,
 
 
 
->tip The __RadTreeListView__ can use the same columns as the __RadGridView__. To learn more about the different types of columns read here.
+>tip __RadTreeListView__ can use the same columns as __RadGridView__. To learn more about the different types of columns read [here](%slug radgridview-columns-column-types-basic-column).
         
 
 >tip To change the column, which holds the tree view UI, use the __HierarchyColumnIndex__ property.
 
 Here is a snapshot of the result.
 
+__Figure 2: RadTreeListView with manually defined columns__
+
 ![](images/RadTreeListView_GettingStarted_01.png)
 
+## RadTreeListView vs RadGridView
+
+__RadTreeListView__ is a control used to display hierarchical data in a tabular format. In order to achieve this, the control combines the functionality of __RadGridView__ control with the ability to display data in a tree view manner. This is implemented by directly inheriting __RadGridView__ and implementing the tree view behavior.
+
+__RadTreeListView__ displays the data in the same way as __RadGridView__. The difference is that each row plays the role of a tree node, which can be expanded or collapsed.
+
+>Please note, that __RadTreeListView__ and the [Hierarchical RadGridView]({%slug gridview-hierachy-overview%}) are two different things. The first displays the data in a tree view manner in one and the same table. The second one displays the hierarchical data in nested __RadGridViews__. 

```

Comment:
Why are all those tips or notes? To me, they should be part of the main content. Consider paraphrasing the sentence about homogeneous data so it sounds like an instruction and not a suggestion - "Use RadTreeListView only when you need to display homogeneous data..." if we're aiming at people doing it always.